tests/unit/lms/views/error_test.py

```diff
@@ -103,45 +103,6 @@ class TestError(ExceptionViewTest):
     }
 
 
-@pytest.mark.usefixtures("pyramid_config")
```

Comment:
We don't need to test this any more. We would just be testing that Pyramid decorators work